### PR TITLE
[10.0][ADD] event_sale_registration_multy_qty: New module

### PR DESCRIPTION
**Pull request by chienandalu**
Event Sale Registration Multi Qty
=================================

This module extends Event Session Registration Multi Qty so it can be used on
event session sales.

**Depends on:**

- [x] event_registration_multi_qty: https://github.com/OCA/event/pull/78

cc @Tecnativa

### CODE REVIEW

**Comment by yajo** [comment]
#78 merged, please rebase

**Comment by chienandalu** [comment]
@Yajo Runbot didn't get the merge yet

**Comment by rafaelbn** [comment]
Hi @chienandalu @Yajo @sergio-teruel rebuilding for testing

**Comment by pedrobaeza** [comment]
@chienandalu can you improve test coverage?

@rafaelbn @Yajo please review

File: event_sale_registration_multi_qty/README.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit https://odoo-community.org.
+
```
Comment (yajo) [inline event_sale_registration_multi_qty/README.rst:53]: :fire:

File: event_sale_registration_multi_qty/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Event Sale Registration Multi Qty',
+    'version': '10.0.1.0.0',
+    'author': 'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    "license": "AGPL-3",
+    'website': 'https://www.tecnativa.com',
```
Comment (yajo) [inline event_sale_registration_multi_qty/__manifest__.py:12]: Use github url

File: event_sale_registration_multi_qty/models/sale_order.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+from odoo import api, models
+
+
+class SaleOrderLine(models.Model):
+
+    _inherit = 'sale.order.line'
+
+    @api.multi
+    def _update_registrations(self, confirm=True, registration_data=None):
+        """ Writes registrations """
```
Comment (yajo) [inline event_sale_registration_multi_qty/models/sale_order.py:15]: Remove surrounding whitespace: https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

File: event_sale_registration_multi_qty/tests/test_event_sale_registration_multi_qty.py [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        sale.action_confirm()
+        reg = self.env['event.registration'].search([
+            ('sale_order_id', '=', sale.id)
+        ])
+        editor = self.env['registration.editor'].create({
+            'sale_order_id': sale.id,
+            'event_registration_ids': reg,
+        })
+        editor.action_make_registration()
+        # self.assertEqual(reg.state, 'open')
```
Comment (yajo) [inline event_sale_registration_multi_qty/tests/test_event_sale_registration_multi_qty.py:91]: :fire:

File: event_sale_registration_multi_qty/models/sale_order.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        Registration = self.env['event.registration']
+        registrations = Registration.search([
+            ('sale_order_line_id', 'in', self.ids)
+        ])
+        for so_line in self.filtered('event_id'):
+            if not so_line.event_id.registration_multi_qty:
+                super(SaleOrderLine, so_line)._update_registrations()
```
Comment (yajo) [inline event_sale_registration_multi_qty/models/sale_order.py:22]: Did you forget to pass args?

**Review by yajo — APPROVED** [review]
Some comments, they don't seem much critical.

Only code review.

**Review by rafaelbn — APPROVED** [review]
:+1:

**Pull request merged**